tests/mol-timing-synchronization_multiple_audio/EPUB/package.opf

```diff
@@ -12,7 +12,8 @@
     <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
     <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
     <meta property="belongs-to-collection">should</meta>
-    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#mol-audio-exceeding-clipend</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#mol-timing-sync</meta>
+    <meta property="dcterms:isReferencedBy">https://www.w3.org/TR/epub-rs-33/#mol-rendering-audio</meta>
     <meta property="dcterms:modified">2022-02-14T00:00:00Z</meta>
     <meta property="media:duration" refines="#md-smil">00:02:47.768</meta>
```

Comment:
I believe this duration value is wrong. 
One day, I will get around to writing a utility that fills this in automatically, although by then, we will probably be done writing MO tests.

Reply:
Sigh... yes, indeed. I think the right value is `00:01:46.35`. I use a [time calculator](http://www.unitarium.com/time-calculator) which, hopefully, is correct.

We are violating the DRY principle here, b.t.w. Wouldn't it be enough to require the duration per smil files and the overall media duration would then be the sum of all those? But anyway, that is a discussion for another day...

Thx!